ArduinoAddons/Arduino_1.0.x/libraries/U8glib/utility/u8g_rot.c

```diff
@@ -40,7 +40,11 @@ uint8_t u8g_dev_rot90_fn(u8g_t *u8g, u8g_dev_t *dev, uint8_t msg, void *arg);
 uint8_t u8g_dev_rot180_fn(u8g_t *u8g, u8g_dev_t *dev, uint8_t msg, void *arg);
 uint8_t u8g_dev_rot270_fn(u8g_t *u8g, u8g_dev_t *dev, uint8_t msg, void *arg);
 
+#ifdef __XC8
 uint8_t u8g_dev_rot_dummy_fn(void *u8g, void *dev, uint8_t msg, void *arg)
+#else
+uint8_t u8g_dev_rot_dummy_fn(u8g_t *u8g, u8g_dev_t *dev, uint8_t msg, void *arg)
+#endif
 {
   return 0;
 }
```

Comment:
Don't patch u8glib!
Better copy in the new version.

>2016-04-05	v1.19	
  * Fixed contrast (brightness) setting for SH1106 and SSD1306 devices
  * Fixed compiler warnings